File: src/main/java/spaceEngineers/commands/SeSessionCommand.java
package spaceEngineers.commands;

public class SeSessionCommand {
    // TODO(PP): Add request type enum. So far only load command is supported.
    public String scenarioPath;

    private SeSessionCommand(String scenarioPath) {

        this.scenarioPath = scenarioPath;
    }

    public static SeSessionCommand load(String scenarioPath) {
        return new SeSessionCommand(scenarioPath);
    }
}

File: src/main/java/spaceEngineers/commands/SeSessionCommand.kt
package spaceEngineers.commands

class SeSessionCommand private constructor(  // TODO(PP): Add request type enum. So far only load command is supported.
    var scenarioPath: String
) {
    companion object {
        fun load(scenarioPath: String): SeSessionCommand {
            return SeSessionCommand(scenarioPath)
        }
    }
}